contracts/crowdsale/validation/WhitelistedCrowdsale.sol

```diff
@@ -0,0 +1,39 @@
+pragma solidity ^ 0.4.18;
+
+import "../Crowdsale.sol";
+import "../../ownership/Ownable.sol";
+
+/**
+ * @dev Crowdsale in which only whitelisted users can contribute
+ */
+contract WhitelistedCrowdsale is Crowdsale, Ownable {
+
+  mapping(address => bool) public whitelist;
+
+  /**
+   * @param _beneficiary Address to be added to the whitelist
+   */
+  function addToWhitelist(address _beneficiary) external onlyOwner {
```

Comment:
Considering this is a _crowd_ sale, we should provide an `addToWhitelist(address[] _beneficiaries)` function, that would save lots of function calls.

Reply:
or a constructor